### DISCUSSION TITLE
How to integrate it with Django Project

### DISCUSSION THREAD

**Discussion by swastkk** [anujdutt9/Disease-Prediction-from-Symptoms#13] [General]
Plz provide information on how to integrate this model with Python-Django Web app.

**Comment by anujdutt9** [comment]
Hi. For Python-Django Web app, you could train your model, save it and make predictions on user input using maybe the Python Flask API.

**Comment by nabinstha1234** [comment]
https://github.com/nabinstha1234/Disease-prediction-with-ml here is implemented link.

**Comment by khanbasharat3a1** [comment]
Same question but with flask.....i want to make a  healthcare chatbot
